src/action_runner/action_runner.vcxproj

```diff
@@ -23,7 +23,7 @@
     <VCProjectVersion>16.0</VCProjectVersion>
     <ProjectGuid>{D29DDD63-E2CF-4657-9FD5-2AEDE4257E5D}</ProjectGuid>
     <RootNamespace>actionrunner</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0.17134.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
```

Comment:
I'm not sure this is ok.  Generally you need to target a specific build of windows.  Please confirm before proceeding.